src/styles/blocks/star.scss

```
@@ -0,0 +1,21 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+  }
+
+}
+
+.stars--1 .stars__star:nth-child(-n+1),
```

use the power of nesting
```suggestion
  &--1 .stars__star:nth-child(-n+1),
}
```